src/main/java/baritone/utils/Helper.java

```diff
@@ -51,6 +44,7 @@
 
     Minecraft mc = Minecraft.getMinecraft();
 
+    /*

```

Comment:
=O

Reply:
LOL yeah should probably just remove it entirely now.